src/doc/imagebufalgo.rst

```diff
@@ -1166,19 +1166,24 @@ Shuffling channels
           ImageSpec hint;
           hint["raw:Demosaic"] = "none";
           ImageBuf Src ("test.cr3", 0, 0, nullptr, &hint);
-          ParamValue options[] = { { "layout", "GRBG" } };
+          float wb[4] = {2.0, 0.8, 1.2, 1.5};
+          ParamValue options[] = {
+                { "layout", "GRBG" },
+                ParamValue("white-balance", TypeFloat, 4, wb)
+          };
```

Comment:
Not requesting this as part of this PR, but just noting that at some point, we should do the thing where we move these actual code snippets to the appropriate parts of testsuite/docs-examples-cpp and testsuite/docs-examples-cpp, and then just include them by reference in the docs, so that it is confirmed that they compile and work correct with every CI run.

Reply:
I can take a look on the weekend. Marking as a draft for now.